packages/manager/modules/vrack/src/translations/Messages_fr_FR.json

```diff
@@ -36,9 +36,11 @@
   "vrack_menu_project_num": "Vrack {{num}}",
   "vrack_service_type_dedicatedserver": "Serveur dédié",
   "vrack_service_type_dedicatedcloud": "Private Cloud",
+  "vrack_service_type_managedbaremetal": "Managed Bare Metal",
   "vrack_service_type_dedicatedconnect": "Dedicated connect",
   "vrack_service_type_legacyvrack": "Vrack legacy",
   "vrack_service_type_dedicatedclouddatacenter": "Private Cloud",
+  "vrack_service_type_managedbaremetaldatacenter": "Managed Bare Metal",
```

Comment:
same translation as vrack_service_type_managedbaremetal ?

Reply:
Yeah... but in this case, we built up the translation id using the datacenter type. Here it is `managedbaremetaldatacenter`. Hence we would not be able to reuse the same translation id